models/quality_measures/intermediate/adh-statins_medication_adherence_for_cholestrol/quality_measures__int_adh_statins_numerator.sql

```diff
@@ -0,0 +1,366 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with denominator as (
+
+    select
+          patient_id
+        , dispensing_date
+        , first_dispensing_date
+        ,30 as days_supply
```

Comment:
@BijayaShres can you explain the reason for populating days_supply with "30"? Is this leftover from testing?

Reply:
Yes @sarah-tuva, that was a leftover from testing. I will get it fixed and ensure not to repeat it moving forward.
Thank you for pointing out.

Reply:
Thanks @BijayaShres! 🙏